src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
You should link CSS files in HTML, not SCSS. Browsers don't understand SCSS syntax, it should be compiled into CSS first. Change the href attribute to 'styles/index.css'.